q2_diversity_lib/citations.bib

```diff
@@ -1,36 +1,157 @@
 @article{faith1992conservation,
-  title={Conservation evaluation and phylogenetic diversity},
-  author={Faith, Daniel P},
-  journal={Biological conservation},
-  volume={61},
-  number={1},
-  pages={1--10},
-  year={1992},
-  publisher={Elsevier},
-  doi={10.1016/0006-3207(92)91201-3}
+  title = {Conservation evaluation and phylogenetic diversity},
+  author = {Faith, Daniel P},
+  journal = {Biological conservation},
+  volume = {61},
+  number = {1},
+  pages = {1--10},
+  year = {1992},
+  publisher = {Elsevier},
+  doi = {10.1016/0006-3207(92)91201-3}
```

Comment:
Please pick a style - `=` or ` = `, either are fine by me, just keep everything consistent (you have both styles in the additions list below).